packages-exp/auth-exp/src/core/user/token_manager.ts

```diff
@@ -42,6 +42,7 @@ export class StsTokenManager {
   refreshToken: string | null = null;
   accessToken: string | null = null;
   expirationTime: number | null = null;
+  isPassthroughMode: boolean = false;
```

Comment:
since this state is equivalent to `this.refreshToken == null` do we need the extra field?

Reply:
Similar to other discussion threads - it's possible that `refreshToken == null` even when we're not in passthrough mode